ferrocene/doc/specification/.github/workflows/ci.yml

```diff
@@ -23,26 +23,20 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      # Some Ferrocene builders require the use of Python 3.9. Use that on CI
-      # to make sure there are no surprises when we import into Ferrocene.
-      - uses: actions/setup-python@v3
-        with:
-          python-version: "3.9.x"
+      - name: Setup uv
+        uses: astral-sh/setup-uv@v5
```

Comment:
Should specify the uv version -- They broke their CLI between 0.4 and 0.5 and we should coordinate future upgrades to avoid new breakage.